website/docs/r/organization_token.html.markdown

```diff
@@ -28,6 +28,28 @@ The following arguments are supported:
 * `force_regenerate` - (Optional) If set to `true`, a new token will be
   generated even if a token already exists. This will invalidate the existing
   token!
+* `expired_at` - (Optional) The token's expiration date. The expiration date must be a date/time string in RFC3339 
```

Comment:
I've updated the description for `expired_at` per your suggestions, but opted to leave the "optional" here since the expiry is optional and seems consistent with the other optional arguments, but will happily remove it if preferred! Additionally, I moved the example usage back to where it initially was (beneath the arguments docs!) to keep it more inline with the expiry for context. 

Let me know what you think, @brandonc! 😄